wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -224,6 +224,10 @@ const SingleRoute = (props: Props) => {
   }, [props.route.name]);
 
   const providerText = useMemo(() => {
+    if (!sourceToken) {
```

Comment:
This is a very rare case (I've seen it when switching chains), but good to be safe.

Reply:
came from here https://github.com/wormhole-foundation/wormhole-connect/commit/5c13b8d1aea40d1ee896deb4c0e7a375cffdbb02#diff-50062b7f121564dcafd21f3ee9350314aae20a3a4b2487eff668042c0d358587R36